addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -1880,9 +1880,8 @@ export class Order extends PosModel {
         }
     }
 
-    initialize_validation_date() {
-        this.validation_date = new Date();
-        this.formatted_validation_date = formatDateTime(DateTime.fromJSDate(this.validation_date));
+    update_date_order() {
```

Comment:
We don't need this. It's only a function with one line.
Just do it directly.

Reply:
Hello @rhe-odoo 
Requested changes are done.